File: java/vetores/Negativos.java
import entities.ExercicioNegativos;

import java.util.Locale;
import java.util.Scanner;

public class Negativos {

    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        int i, N, num;

        System.out.print("Quantos numeros voce vai digitar? ");
        N = sc.nextInt();

        ExercicioNegativos[] numeros = new ExercicioNegativos[N];

        for(i = 0; i < N; i++) {
            System.out.print("Digite um número: ");
            num = sc.nextInt();
            numeros[i] = new ExercicioNegativos(num);
        }

        System.out.println();
        System.out.println("NUMEROS NEGATIVOS: ");
        for(i = 0; i < N; i++) {
            if (numeros[i].getNum() < 0) {
                System.out.println(numeros[i].getNum());
            }
        }

        sc.close();
    }
}

File: js/vetores/negativos.js
const readline = require('readline-sync');

const X = Number(readline.question("Quantos numeros voce vai digitar? "));
const numeros = [];

for (let i = 0; i < X; i++) {
    const num = Number(readline.question("Digite um numero: "));
    numeros.push(num);
}

console.log("NUMEROS NEGATIVOS:");
for (let i = 0; i < X; i++) {
    if (numeros[i] < 0) {
        console.log(numeros[i]);
    }
}